src/get_manifest.ts

```diff
@@ -0,0 +1,128 @@
+import { deepMerge, getProtocolInterface, path } from "./deps.ts";
```

Comment:
Similar to `build.ts`, this file encapsulates the old deno-slack-builder's `manifest.ts` file. It implements the `get-manifest` hook.